documentation/docs/20-core-concepts/60-remote-functions.md

```diff
@@ -450,6 +450,42 @@ Alternatively, you could use `select` and `select multiple`:
 
 > [!NOTE] As with unchecked `checkbox` inputs, if no selections are made then the data will be `undefined`. For this reason, the `languages` field uses `v.optional(v.array(...), [])` rather than just `v.array(...)`.
 
+### Imperative validation
+
+In addition to declarative schema validation, you can imperatively mark fields as invalid inside the form handler using the `invalid` function. This is useful for cases where you can't know if something is valid until you try to perform some action:
+
+```js
+/// file: src/routes/shop/data.remote.js
+import * as v from 'valibot';
+import { form } from '$app/server';
+import * as db from '$lib/server/database';
+
+export const buy_hotcakes = form(
```

Comment:
reluctantly I think we should use camelCase in docs (though I'm sure there are places where we're inconsistent)

```suggestion
export const buyHotcakes = form(
```